addons/hr/data/hr_demo.xml

```diff
@@ -7,6 +7,23 @@
                 (3, ref('hr.group_hr_manager'))]"/>
         </record>
 
+        <!-- Resource Calendar -->
+        <record id="resource_calendar_std_20h" model="resource.calendar">
+            <field name="name">Standard 20 hours/week</field>
+            <field name="company_id" eval="False"/>
```

Comment:
Why is it needed to set no company?

Reply:
Setting company to False ensures that the resource calendar is available across all companies in a multi-company setup. If a specific company is assigned, the resource calendar will only be accessible within that company. If no company is explicitly set, by default the current company is automatically assigned. This would restrict access to other companies in a multi-company environment. To make the resource calendar universally available, the company is set to False.

Reply:
@desh-odoo okay, I know that but the resource calendar will really be used in more than one company?

Reply:
> @desh-odoo okay, I know that but the resource calendar will really be used in more than one company?

We are doing it like the `standard 38 hours/week` calendar. Right now, we have used it for one case, but in the future, anyone can use it.

Reply:
okayyyy let's keep as you did.